CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing guidelines
+
+Please follow the same [contributing guide](https://github.com/kubevirt/community/blob/master/contributers/contributing.md) as you would follow for any other pull request made to Kubevirt.
```

Comment:
FWIW: this link is a 404.  I see a kubevirt/community github repo, but I don't see anything like a "contributors" or "contributing.md" file, so I can't offer a PR to know where you want this link to point.

Reply:
I have this [PR](https://github.com/kubevirt/community/pull/89) in progress in Community that will resolve the 404